.github/workflows/docker_publish.yml

```
@@ -215,9 +215,11 @@ jobs:
     name: Running Housekeeping scrips
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: recursive
+          set-safe-directory: "/__w/DAGMC/DAGMC"
```

Is this supposed to work?  It's essentially what I did in #807 
```suggestion
          set-safe-directory: ${GITHUB_WORKSPACE}
```